wildfly-modules/src/main/resources/org/infinispan/server/main/module.xml

```diff
@@ -16,6 +16,7 @@
       <module name="org.infinispan" slot="${slot}"/>
       <module name="org.infinispan.tasks" slot="${slot}" />
       <module name="org.infinispan.tasks.api" slot="${slot}" />
+      <module name="org.infinispan.query" slot="${slot}" />
```

Comment:
I'm afraid this module is not used by the Infinispan Server, but for the Infinispan modules to be installed on top of Widlfy. This will probably fix the Wildfly + Modules use case, but not the server.

Reply:
The server module is https://github.com/infinispan/infinispan/blob/master/server/integration/feature-pack/src/main/resources/modules/system/layers/base/org/infinispan/server/main/module.xml